vnext/Microsoft.ReactNative/packages.config

```diff
@@ -8,5 +8,5 @@
   <package id="Microsoft.Windows.CppWinRT" version="2.0.200615.7" targetFramework="native" />
   <package id="Microsoft.WinUI" version="3.0.0-preview2.200713.0" targetFramework="native"/>
   <!-- package id="ReactNative.Hermes.Windows" version="0.5.0-f56606a8" targetFramework="native" / -->
-  <!-- package id="ReactNative.V8Jsi.Windows" version="0.3.3" targetFramework="native" / -->
+  <!-- package id="ReactNative.V8Jsi.Windows.UWP" version="0.3.4" targetFramework="native" / -->
```

Comment:
>Windows [](start = 37, length = 7)

(nit) remove commented out lines?

Reply:
We want to make it very easy to experiment with Hermes in RNW, uncommenting a line is easier that finding a (potentially out of date) documentation on which package and version to include.